frontend-react/src/App.tsx

```diff
@@ -90,127 +89,129 @@ const App = () => {
             onAuthRequired={customAuthHandler}
             restoreOriginalUri={restoreOriginalUri}
         >
-            <Suspense fallback={<Spinner size={"fullpage"} />}>
-                <NetworkErrorBoundary
-                    fallbackComponent={() => <ErrorPage type="page" />}
-                >
-                    <DAPHeader env={process.env.REACT_APP_ENV?.toString()} />
-                    <GovBanner aria-label="Official government website" />
-                    {context.values.org && context.values.senderName ? (
+            <SessionProvider>
```

Comment:
This has to wrap _within_ the Security tag to make use of `useOktaAuth()` That said, perhaps that Security tag should move up to the `index.tsx` level.